elfpy/utils/math/fixed_point.py

```diff
@@ -116,16 +149,27 @@ def __floordiv__(self, other: int | FixedPoint) -> FixedPoint:
         other = self._coerce_other(other)
         if other is NotImplemented:
             return NotImplemented
-        if other <= FixedPoint("0.0"):
```

Comment:
was this an error before? should we have tests to catch it?

Reply:
I think it was checking <0 before because we copied the solidity contracts which generally use unsigned ints. But we're living in a signed int world so we don't care if it is negative.